BUG: Link libcgset with libcgroup

drakenclimber opened the pull request:
When linking with the -Wl,--no-undefined flag enabled, libcgset
fails due to undefined symbols.  Fix this by enabling this flag
and linking with libcgroup.la

Closes: https://github.com/libcgroup/libcgroup/issues/66
Signed-off-by: Tom Hromatka <tom.hromatka@oracle.com>

kamalesh-babulal commented on src/tools/Makefile.am:
```diff
@@ -27,6 +27,8 @@ cgcreate_CFLAGS = $(CODE_COVERAGE_CFLAGS)
 libcgset_la_LIBADD = $(CODE_COVERAGE_LIBS)
 libcgset_la_CFLAGS = $(CODE_COVERAGE_CFLAGS) -DSTATIC= -DUNIT_TEST
+libcgset_la_LDFLAGS = -Wl,--no-undefined $(top_builddir)/src/libcgroup.la \
+		      -lpthread
```
Looks good to me. Just a style preference comment, can it be simplified more by using the LADD? using libcgset_la_LDFLAGS = -Wl,--no-undefined $(LDADD)

drakenclimber replied:
Good idea.  Thanks, Kamalesh.

drakenclimber commented:
Fixed in main in commit https://github.com/libcgroup/libcgroup/commit/b4015e3f96fc430029a37e94edc151e16727e8f1
Fixed in release-2.0 in commit https://github.com/libcgroup/libcgroup/commit/e34df2f6614eb107ba3883ab270ab51a69ca0515

Thanks all!

mikhailnov commented:
Thanks!
